exs/mundo_1/java/023.java

```diff
@@ -0,0 +1,23 @@
+/*
+    ### 023 - Separando dígitos de um número
+    Faça um programa que leia um número de 0 a 9999 e mostre na tela cada um dos dígitos separados
+ */
+
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        String numero;
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Digite um numero: ");
+        numero = scanner.nextLine();
+
+        for (char n : numero.toCharArray()) {
+            System.out.print(n + " ");
+        }
+    }
+
+}
```

Comment:
A solução desse exercício está incorreta. 

Compreendo o motivo de ter solucionado como o fez, pois o enunciado da realmente a enteder que é necessário apenas separar os digitos com um espaço.

**A forma correta é a separação por casas númericas (unidade, dezena, centena e milhar) e não por espaços entre os dígitos.**

Você pode conferir uma das soluções já realizadas para ter uma base de como solucionar em Java, como, por exemplo, a [solução feita em Python](https://github.com/Coding-in-community/exercicios-CeV/blob/master/exs/mundo_1/python/023.py). E caso ainda tenha dúvida, pode conferir a [aula do exercício 023 no Youtube](https://www.youtube.com/watch?v=wD2aerLMBWA).